shared/web/files.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package web
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var UnsafeContentTypes = [...]string{
```

Comment:
Let's remove this and `MediaContentTypes` from the api4 package

Reply:
Okay